doc/sortable.md

```diff
@@ -304,24 +304,21 @@ $item2->setPosition(-1);
 Sortable works by comparing objects in the same group to see how they should be positioned. From time to time you may want to customize the way these
 objects are compared by simply implementing the Doctrine\Common\Comparable interface
 
+**Warning** the `Comparable` interface should be implemented on the group object
+
 ``` php
 <?php
 namespace Entity;
 
 use Doctrine\Common\Comparable;
 
 /**
- * @ORM\Table(name="items")
- * @ORM\Entity(repositoryClass="Gedmo\Sortable\Entity\Repository\SortableRepository")
+ * @ORM\Table(name="category")
  */
-class Item implements Comparable
+class Category implements Comparable
 {
     public function compareTo($other)
     {
-        // return 1 if this object is considered greater than the compare value
```

Comment:
Why we should remove these comments?
The description at [`Doctrine\Common\Comparable`](https://github.com/doctrine/common/blob/3f7b18fb66f2cf65cc7e170ad321c1272b643efc/lib/Doctrine/Common/Comparable.php#L15-L16) states the following:
>Returns 0 if they are semantically equal, 1 if the other object is less than the current one, or -1 if its more than the current one.

Reply:
Because at [this line](https://github.com/doctrine-extensions/DoctrineExtensions/blob/ea1e0253f5c148754a525865c7277f38396033e9/src/Sortable/SortableListener.php#L467), the comparison is done against a _truthy_ value, and `if (-1)` would evaluate to `true`. So if you return literally _anything else_ than 0, the objects would be considered as different. 

This is what I explained in the pull request initial comment, we don't want to compare objects, we want to guess if they are equal. 
> Actually, we are looking for a isEquatable interface.

Reply:
Well, even if I guess the loose comparison against `-1` is a valid reason by now, please be aware that this kind of condition will be surely replaced by its strict variant in the next releases.
As I'm not using this extension, I thought the possible values returned by the method could be used to determine the position in the sorted result, using a similar logic than the provided by the `<=>` operator.